src/@shared/DashboardHeader.tsx

```diff
@@ -7,14 +7,13 @@ export function DashboardHeader() {
   const { userName, projectName } = useParams();
   const location = useLocation();
   const { pathname } = location;
+  const regExp = /\/([^/]+)\/([^/]+)\/([^/]+)$/;
```

Comment:
```suggestion
  const [userName, projectName, currentDashboard] = location.split("/");
```

regExp대신 이렇게 해줄 수도 있지 않을까요..?